llm/ext_server_common.go

```diff
@@ -1,7 +1,7 @@
 package llm
 
 /*
-#cgo CFLAGS: -I${SRCDIR}/llama.cpp/gguf -I${SRCDIR}/llama.cpp/gguf/common -I${SRCDIR}/llama.cpp/gguf/examples/server
+#cgo CFLAGS: -I${SRCDIR}/llama.cpp -I${SRCDIR}/llama.cpp/gguf -I${SRCDIR}/llama.cpp/gguf/common -I${SRCDIR}/llama.cpp/gguf/examples/server
```

Comment:
not important for this PR but we could simplify the directory structure a bit – ideally we have a llama.cpp `go` package with the accompanying submodule (e.g. there's no need for `gguf` anymore)

Reply:
Good call out.  I'll play around with options on restructuring the directories in a follow up to see if we can simplify things more.